### ISSUE TITLE
Citizen Integrator Lab 4 Step 3.1 - screenshot does not match instructions

### ISSUE THREAD

**Issue by scranton** [RedHatWorkshops/dayinthelife-integration#108]

**Comment by scranton** [comment]
Need to verify format of all 3 URLs

**Comment by sigreen** [comment]
Assigning to vinay, as all the 3scale URL's need to be updated throughout this lab

**Comment by VinayBhalerao** [comment]
Blocked on https://github.com/RedHatWorkshops/dayinthelife-integration/issues/131

**Comment by VinayBhalerao** [comment]
The lab is updated with new screenshots and updated instructions https://github.com/RedHatWorkshops/dayinthelife-integration/commit/501b7e553d4e406648b77aebb80311ff9cb0db0d

Note: The instructions are updated to use the details from #131. We can create a new issue if the urls changes. 

Closing.
